src/commands/get-user-tokens.js

```diff
@@ -9,13 +9,14 @@ Cypress.Commands.add('getUserTokens', (credentials = defaultCredentials) => {
   const { username, password } = credentials;
 
   return new Cypress.Promise((resolve, reject) => {
-    auth.client.loginWithDefaultDirectory(
+    auth.client.login(
       {
         username,
         password,
         audience: Cypress.env('auth0Audience'),
         scope: Cypress.env('auth0Scope'),
-        client_secret: Cypress.env('auth0ClientSecret'), // eslint-disable-line @typescript-eslint/camelcase
+        client_secret: Cypress.env('auth0ClientSecret'), // eslint-disable-line @typescript-eslint/camelcase,
+        realm: Cypress.env('auth0Realm') || 'Username-Password-Authentication',
```

Comment:
I haven't run the tests yet, since we need to add the correct `env` variables but this approach seems sane to me. 👍